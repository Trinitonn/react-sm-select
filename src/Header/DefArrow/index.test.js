import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { DefArrow } from './index';

describe('DefArrow', () => {
  it('should render properly', () => {
    const wrapper = shallow(
      <DefArrow
        expanded={true}
        disabled={true}
      />
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should render properly if noe expanded - disables', () => {
    const wrapper = shallow(
      <DefArrow
        expanded={false}
        disabled={false}
      />
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
