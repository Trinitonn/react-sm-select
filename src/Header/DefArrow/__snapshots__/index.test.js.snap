// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefArrow should render properly 1`] = `
<div
  className="Header__arrow"
>
  <div
    className="Header__arrow--up"
  />
</div>
`;

exports[`DefArrow should render properly if noe expanded - disables 1`] = `
<div
  className="Header__arrow"
>
  <div
    className="Header__arrow--down"
  />
</div>
`;
